scripts/webpack/proxyPlugin.js

```diff
@@ -117,7 +117,13 @@ class ProxyPlugin {
 
       req
         .pipe(
-          request(cleanProxyUrl + req.url).on('response', (response) => {
+          request({
+            url: cleanProxyUrl + req.url,
+            rejectUnauthorized: false,
+            followAllRedirects: false,
+            followRedirect: false,
+            followOriginalHttpMethod: true,
```

Comment:
Necessary to support login redirects after OIDC login. Doesn't break anything else (so far)